tvaultuiv2/src/views/private/safe/components/Tabs/index.js

```diff
@@ -198,7 +201,7 @@ const SelectionTabs = (props) => {
           });
         });
     }
-  }, [safeDetail]);
+  }, [safeDetail, history.location.state]);
```

Comment:
Why "**history.location.state**" is a dependency? We are not relying on location's state